src/aims_ui/models/get_fields.py

```diff
@@ -24,18 +24,18 @@ def get_fields(endpoint_name):
           search_type='radio',
           flag=False,
           display_title='Epoch',
-          default_radio_selection='84',
+          default_radio_selection='85',
```

Comment:
Available epochs are 87, 85, 80 and 39. 85 is the default but will flip to 87 later